xbmc/video/VideoInfoScanner.cpp

```diff
@@ -472,10 +472,16 @@ namespace VIDEO
           mediaType = MediaTypeTvShow;
         else if (info2->Content() == CONTENT_MUSICVIDEOS)
           mediaType = MediaTypeMusicVideo;
-        CServiceBroker::GetEventLog().Add(EventPtr(new CMediaLibraryEvent(
-          mediaType, pItem->GetPath(), 24145,
-          StringUtils::Format(g_localizeStrings.Get(24147).c_str(), mediaType.c_str(), URIUtils::GetFileName(pItem->GetPath()).c_str()),
-          pItem->GetArt("thumb"), CURL::GetRedacted(pItem->GetPath()), EventLevel::Warning)));
+
+        if (info2->Content() == CONTENT_TVSHOWS)
+          CServiceBroker::GetEventLog().Add(EventPtr(new CMediaLibraryEvent(
+            mediaType, pItem->GetPath(), 24145,
+            StringUtils::Format(g_localizeStrings.Get(24147).c_str(), mediaType.c_str(),CURL::GetRedacted(pItem->GetPath())), EventLevel::Warning)));
+        else if (info2->Content() == CONTENT_MOVIES || info2->Content() == CONTENT_MUSICVIDEOS)
+          CServiceBroker::GetEventLog().Add(EventPtr(new CMediaLibraryEvent(
+            mediaType, pItem->GetPath(), 24145,
+            StringUtils::Format(g_localizeStrings.Get(24147).c_str(), mediaType.c_str(), URIUtils::GetFileName(pItem->GetPath()).c_str()),
+            pItem->GetArt("thumb"), CURL::GetRedacted(pItem->GetPath()), EventLevel::Warning)));
```

Comment:
Any reason you don't pass the thumb art and the path for tvshows?

Reply:
I've never seen thumbs working and AFAICT, it also doesn't work for movies even if it's passed for them. For tvshows the path is in use. I dropped the filename.